### DISCUSSION TITLE
Include more frames when training dp potential model?

### DISCUSSION THREAD

**Discussion by hu-lm** [deepmodeling/deepmd-kit#1912] [Q&A] [answered]
Hi,

I am using deepmd version 2.1.3. I have a question when training the dp model.

Is it possible to include more frames during the training process, or for example, continue training based on a new dataset with more frames/snapshots while using an existing dp model? Or is there an easier way to add more frames into the dataset during the training process? Thank you so much!

**Comment by wanghan-iapcm** [comment, marked as answer]
You may use `--init-model` with the `dp train` command to training a new model from the model parameters of an old model.  Not that you may reducing the learning rate so the information stored in the old model would not been forgotten, while increase the relative possibility of visiting the new data systems (by default the possibility is in proportional to the size of the data systems.)

Here is an example, please note that the start learning rate is set to 0.0001 and the  possibility of using the new data systems is 30%

```
"learning_rate" :{
        "type":         "exp",
        "start_lr":     0.0001,  
        "stop_lr":      3.51e-8,
        "decay_steps":  5000,
},

"training" : {
"training_data": {
    "systems" : [ "old-sys-0", "old-sys-1", "old-sys-2", "new-sys-0", "new-sys-1"],
    "auto_prob_style" : "prob_sys_size;0:3:0.7;3:5:0.3"
},
...
}
```
